ksp_plugin_adapter/flight_planner.cs

```diff
@@ -67,7 +67,7 @@ class FlightPlanner : SupervisedWindowRenderer {
     // The UI code proper, executed identically for Layout and Repaint.  We
     // can freely change the state in events like clicks (e.g., in if statements
     // for buttons) as these don't happen between Layout and Repaint.
-    string vessel_guid = vessel_?.id.ToString();
+    string? vessel_guid = vessel_?.id.ToString();
     if (vessel_guid == null || !plugin.HasVessel(vessel_guid)) {
```

Comment:
put `vessel_ == null` in this conditional, so that we don't need the `!` in `FlightPlanCreate` below.

If we want to avoid repeating null checks or `.id.ToString()`, one option is
```C#
string vessel_guid;
if (vessel_ == null || !plugin.HasVessel(vessel_guid = vessel.id.ToString())) {
  return;
}
```
which convinces the compiler that `vessel` is non-null and `vessel_guid` is initialized after the `if` statement.